lib/burnside/convert-expr.cc

```diff
@@ -412,6 +426,11 @@ class ExprLowering {
     case Ev::LogicalOperator::Neqv:
       result = createCompareOp<M::CmpIOp>(op, M::CmpIPredicate::ne);
       break;
+    case Ev::LogicalOperator::Not:
+      // LogicalOperations are binary operations. Expr for Not is
+      // evaluate::Not<KIND>.
+      assert(false);
```

Comment:
Unless this code is temporary, why not write the assert link on line 417/436?

Reply:
You mean that this assert is useless given the one line 436 ? If so, sure, I will remove the new assert.

Reply:
No, I mean an assert like assert(false && "this is why we have this assert");

Reply:
Thanks for clarifying. It is nicer indeed, will do.